src/main/java/com/google/api/codegen/transformer/ImportTypeTransformer.java

```diff
@@ -51,6 +59,39 @@
       fullNames.add(namer.getProtoFileImportFromService(targetInterface));
     }
 
+    return createImportFilesFromNames(fullNames);
+  }
+
+  public List<ImportTypeView> generateProtoFileImports(SurfaceTransformerContext context) {
```

Comment:
This seems to be way more complicated than what I thought. Why can't we simply use

service.getFile().getSimpleName().replace(".proto", "")

to get the required proto classes? I thought xxx_services_pb.rb has a 1:1 corresponding xxx_pb.rb